### DISCUSSION TITLE
Adding preset possibility for Extras tab in WebUI

### DISCUSSION THREAD

**Discussion by Woisek** [Gerschel/sd_web_ui_preset_utils#33] [Ideas]
In the title.
I find myself more and more having different upscaler combinations for different upscaling tasks. It's getting hard to memorize these over a lenghty period of time.
So, it would be awesome, if we could have presets for all this options, as well, please. :)

**Comment by Gerschel** [comment]
I know a way. At the moment, that work sprint I mentioned in the other chat, goes until at least March 15th now.

**Reply by Woisek** [reply]
OK, I'm happy you will consider this request. Just take your time, we are happy to get it. :)
Thanks in advance!
